src/bundle/DependencyInjection/EzPlatformAdminUiExtension.php

```diff
@@ -41,10 +41,30 @@ public function load(array $configs, ContainerBuilder $container)
      * @param ContainerBuilder $container
      */
     public function prepend(ContainerBuilder $container)
+    {
+        $this->prependViews($container);
+        $this->prependImageVariations($container);
+    }
+
+    /**
+     * @param ContainerBuilder $container
+     */
+    private function prependViews(ContainerBuilder $container): void
```

Comment:
Just curiosity, what does `void` at the end means? This is something new for me.

Reply:
This hints that the method won't return any value. Can be omitted but we enforce type hinting everywhere.

Reply:
That's interesting. It seems you're not using it in other places in this file.
Thanks, good to know.